src/commands/database-remove.js

```diff
@@ -8,14 +8,17 @@ var api = require("../api");
 var responseToError = require("../responseToError");
 var FirebaseError = require("../error");
 
-var utils = require("../utils");
-var prompt = require("../prompt");
+var utils = require("../lib/utils");
```

Comment:
This should not be including from `lib`. You probably want `require("../utils")`.